rr/config/.rr.yaml

```diff
@@ -0,0 +1,20 @@
+http:
+  address: :8080
+  workers:
+    command: "php /var/www/app/worker.php"
```

Comment:
is it a file we need to setup?

Reply:
Yes, it's need to be implemented in Symfony app

Reply:
Is it /app/public/index.php?

All apps run in /app by the way, not /var/app